lib/DateTime.vala

```diff
@@ -68,6 +99,16 @@ namespace Granite.DateTime {
     }
 
     /**
+     * Compare two {@link GLib.DateTime} and @return true if they occur on the same day of the same year
+     */
+    public bool is_same_day (GLib.DateTime day1, GLib.DateTime day2) {
+        if (day1.get_day_of_year () == day2.get_day_of_year () && day1.get_year () == day2.get_year ()) {
```

Comment:
This function could as well be:
`return day1.get_day_of_year () == day2.get_day_of_year () && day1.get_year () == day2.get_year ()`.